src/BlockchainTestsFiller/bcRandomBlockhashTest/randomStatetest70BCFiller.json

```diff
@@ -51,32 +51,6 @@
 				}
 			    }
 		   }
-	   },
-	   {
-		    "network" : ["Byzantium"],
```

Comment:
I copied the expectations from the filled tests postState long time ago. it is just so many tests to write an expect section. also you dont really know what to expect from a random test